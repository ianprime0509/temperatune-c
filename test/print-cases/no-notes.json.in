{
  "name": "Bad example",
  "source": "me"
}
