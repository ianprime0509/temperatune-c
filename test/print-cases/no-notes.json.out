temperatune: octave base name not found
